src/components/content/privacy-wrapper.tsx

```diff
@@ -84,6 +87,7 @@ export function PrivacyWrapper({
       provider: provider,
     })
     if (isTwingle && showIframe) return null
+    if (type == 'h5p' && showIframe) return null
```

Comment:
@Entkenntnis  can we talk about `==` vs. `===` in our codebase sometime?

Reply:
Good point about typescript. Since the compiler should catch those it's mostly a styling thing, but most ppl seem to lean towards still using `===` (ts docs does so for example).
I'll open a PR and try the rule.

Reply:
https://github.com/serlo/frontend/pull/2180